roles/ami_config/tasks/ami-rh-container/ami-rh-container-lab.yml

```diff
@@ -193,14 +185,35 @@
   become: true
 
 - name: Run the script and cache the images
-  command: sudo /home/ec2-user/start-oc.sh
-  become: true
+  shell: /home/ec2-user/start-oc.sh
+  async: 600
+  poll: 0
+  register: start_oc
 
 - name: Grab podman and builda repos
   get_url: 
     url: https://copr.fedorainfracloud.org/coprs/baude/Upstream_CRIO_Family/repo/epel-7/baude-Upstream_CRIO_Family-epel-7.repo
     dest: /etc/yum.repos.d/builda.repo
 
+- name: Install podman packages for the "{{ ami_config }}" AMI
+  yum:  
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - podman
```

Comment:
let's add the same version here for podman as the version used for buildah below

Reply:
done